transaction-relayer/src/main.rs

```diff
@@ -644,3 +670,23 @@ fn refresh_address_lookup_table(
 
     Ok(())
 }
+
+#[derive(Default, Deserialize, Clone)]
+pub struct StakedNodesOverrides {
```

Comment:
what if you import solana-validator crate where this is defined?

Reply:
just felt weird to pull in the whole solana-validator crate just for 1 struct

but probably still better than copy paste

Reply:
i think copy-paste better than new dependency assuming small amount of code like this

Reply:
crate importing nice since we want it to be api compatible with current validator version (if they upgrade api then we get that)

Reply:
don't feel super strongly either way